src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -314,6 +314,24 @@ var form_handlers = function (el) {
         dependency.closest('.form-group').find('input[name=' + dependency.attr("name") + ']').on("dp.change", update);
     });
 
+    $("input[name$=vat_id][data-countries-in-eu]").each(function () {
```

Comment:
This selector actually broke a form in the invoice settings, so I refined it a little